test-cases/tests/installation/a30-validate-installation-of-rhoam-addon-and-integration-with-ld.md

```diff
@@ -9,6 +9,7 @@ products:
       - osd-fresh-install
     targets:
       - 1.0.0
+      - 1.0.1
```

Comment:
Should this also include 1.1.0?

Reply:
thanks for pointing that out, I will actually update this test case to be automatically included every release